packages/semantic-model/src/convert.ts

```diff
@@ -40,7 +41,9 @@ export function convertToSemanticModel(
   reduce(
     sortedLibs,
     (model, { libraryName, fileContent }) => {
-      if (isLibraryFile(libraryName, fileContent, strict)) {
+      if (
+        isLibraryFile(libraryName, fileContent, strict, printValidationErrors)
```

Comment:
all these state parameters make me re-think if we may want to convert these to class **state** instead of a pure functional flow (in the future), particularly if we find ourselves adding more flags.

It may also be possible to pass the flags under a single "complex" object to reduce complexity (if we will need more flags...)

Reply:
I think an object for the options makes sense